src/commands/init/init.py

```diff
@@ -1,15 +1,49 @@
 import glob
 import shutil
 from pathlib import Path
-from typing import Any, Type
+from typing import Any, List, Optional, Type
 
 from git.exc import InvalidGitRepositoryError
 from git.repo import Repo
 
-from src.utils import log_color_provider, VersionManager
+from src.cli import Command
+from src.utils import VersionManager, log_color_provider
 from src.utils.config.project import Project, ProjectConfig
 
 
+class InitCommand(Command):
+    def __init__(self, script_root: Path, version_manager: VersionManager) -> None:
+        super().__init__()
+        self._script_root = script_root
+        self._version_manager = version_manager
+
+    @property
```

Comment:
Those could be defined as properties on the class, right? We don't really do any dynamic calculation in command definition

Reply:
Or perhaps a part of those properties, definition would be more terse as a result

Reply:
I couldn't find a way to create `@abstractproperty` that is not a method. 
The alternative approach is to pass all information through `super().__init__(...)`.

Reply:
I actually like this approach, maybe is a bit `Java`, but quite readable